sys/flash_map/include/flash_map/flash_map.h

```diff
@@ -52,6 +52,15 @@ struct flash_area {
     uint32_t fa_size;
 };
 
+struct sector_range {
+    struct flash_area sr_flash_area;
+    uint32_t sr_range_start;
+    uint16_t sr_first_sector;
+    uint16_t sr_sector_count;
+    uint32_t sr_sector_size;
+    uint8_t sr_align;
```

Comment:
I suspect it's easy to miss filling this in. Would it be better to look up the flash alignment from sr_flash_area device instead?

Reply:
I decided against. I know that this field could be easily forgotten. I did it. But it is filled in one place only and accessed many times so calling function each time even though value is know all the time makes me unhappy.